packages/website/src/theme/MDXComponents/TryInPlayground.tsx

```diff
@@ -2,23 +2,39 @@ import Link from '@docusaurus/Link';
 import clsx from 'clsx';
 import React from 'react';
 
+import { fileTypes } from '../../components/options';
 import styles from './TryInPlayground.module.css';
 
+const fileExtensionsSortedByLength = fileTypes
+  .toSorted((a, b) => b.length - a.length)
+  .map(fileExt => fileExt.replaceAll(/^\./g, ''));
```

Comment:
[Refactor] Nit: this way we don't have to recreate a `new RegExp` each time later on...

```suggestion
  .map(fileExtension => {
    const language = fileExtension.replaceAll(/^\./g, '');
    return [language, new RegExp(`^${fileExtension}\\b`)] as const;
  });
```

```ts
    for (const [fileExtension, tester] of fileExtensionsSortedByLength) {
      if (tester.test(language)) {
```

Reply:
Nice catch! Thanks